awscli/examples/application-insights/create-application.rst

```diff
@@ -0,0 +1,22 @@
+**To create an application from a resource group**
+
+This example uses the ``create-application command`` to create an application from a resource group. ::
```

Comment:
```suggestion
The following ``create-application`` creates an application from a resource group. ::
```
A few of your examples have a typo where the word "command" was in the code formatting. When fixing the formatting in all your intro lines, please check for this fix as well.